cypress/fixtures/articles_data.json

```diff
@@ -0,0 +1,33 @@
+{
+  "articles": [
```

Comment:
here you are receiving the articles in the right way, but this is not how you send them from the backend in the recent PR, please make sure that you have good communication between the teams